app/main.py

```diff
@@ -1,2 +1,60 @@
+from typing import Any
+
+
+class Node:
+    def __init__(self, key: Any, value: Any) -> None:
+        self.key = key
+        self.value = value
+        self.hash = hash(key)
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.load_factor = 2 / 3
```

Comment:
The load factor calculation uses integer division, which results in 0. It should be a float division to correctly represent the load factor as 0.6666.